website/snippets/_new-sl-setup.md

```diff
@@ -107,6 +114,15 @@ To re-enable the dbt Semantic Layer setup in the future, you will need to recrea
 
  The following are the additional flexible configurations for Semantic Layer credentials.
 
+### Map service tokens to credentials
+- After configuring your environment, you can map additional service tokens to the same credential if you have the required permissions.
```

Comment:
Hiya @mirnawong1 

Would it be worthwhile linking out to the permissions page so that users can see what permissions are needed (https://docs.getdbt.com/docs/dbt-cloud-apis/service-tokens#permissions-for-service-account-tokens)

Kind Regards

Reply:
we can! this is the right permissions link though: https://arc.net/l/quote/boxtmheh so I'll add that!